src/pages/LandingPage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import CategoriesBar from '../components/CategoriesBar'
+import '../assets/styles/pages/landing-page.css'
+import illustrationH from '../assets/images/study.svg'
+import illustrationD from '../assets/images/tutors.svg'
+
+class LandingPage extends React.Component {
+
+    render(){
+        return(
+            <div className="BaseContainer">
+                <div className="mockH">Header</div>
+                <section className="LPContainer">
+                    <section className="HeroContainer">
+                        <section className="HeroTextContainer">
+                            <p className='Title'>For each student, <span className="SecondLine">a <span id="Highlight">commited</span> tutor.</span></p>
+                            <p className='Subtitle'>For each question. <span className="SecondLine">The right answer.</span></p>
+                            <div className="Line"></div>
+                        </section>
+                        <img className="Illustration" src={illustrationH} alt="hero illustration" />
+                    </section>
+                    <section className="DescriptionContainer">
+                        <div className="DescriptionTextContainer">
+                            <p className="Title">Ask <span id="Highlight">anything</span>, <span className="SecondLine">we've got you covered</span></p>
+                            <p className="Subtitle">Search for every subject you can imagine, a spetialized tutor will help you with your problem</p>
+                        </div>
+                        <img className="Illustration" src={illustrationD} alt="description illustration" />
+                    </section>
```

Comment:
HeroContainer and DescriptionContainer section are the same, so you can create one component and pass different attributes to render it

Reply:
Noted, just uploaded the LandingPageDescription that puts the HeroContainer and DescriptionContainer in one component.